src/adapter/src/coord/sequencer.rs

```diff
@@ -1459,6 +1488,18 @@ impl Coordinator {
             if_not_exists,
         } = plan;
 
+        if !self
+            .catalog
+            .state()
+            .is_system_schema_specifier(&name.qualifiers.schema_spec)
```

Comment:
Wait, this is so weird. We don't use _system_ IDs for these indexes? Huh.

Reply:
Yep.